ci/docker/gradle-cache-node-github/image.yaml

```diff
@@ -0,0 +1,2 @@
+registry: DOCKER_REGISTRY
+image: android/gradle-cache-node-github
```

Comment:
I'd want to avoid a specific image for an application. It's generic and versionable, but not related to the app.

Reply:
i want it too, but configuration is still a part of an image and i can't find another method to specify it
Tags are also taken by out "hash" publication mechanism, so only image name left

Reply:
> but configuration is still a part of an image

Do you mean `config.yaml`? Can't we mount it?

Reply:
yes, config.yaml
It applies on container start

Reply:
i don't mind two images, honestly

Reply:
Still don't understand why use two different images if all the difference can be customized in `docker run` command.

Reply:
Also, why official image `gradle/build-cache-node` didn't fit the purpose?

Reply:
![image](https://user-images.githubusercontent.com/1105133/122793744-39330500-d2c4-11eb-897b-c870530d1450.png)

phew, finally executed some intiContainer-shell kung-fu to make it work

Reply:
I think such implementation with printf is not worth it. I was hoping to apply a config from file like in docker run.

Reply:
how so? we should either download file in init-container or place it somewhere in cluster to mount.
But persistent mount volumes is not enabled right now.

We need PVC for cache node though, it enables us to do updates without losing cache.
Let's discuss it on another sync

I think it's ok to left only UI configuratble settings (it's like 2 values really) if we'll have PVC, but for now i see `printf` as a valuable compromise (easy installation, easy enough configuration)

Reply:
> how so? we should either download file in init-container or place it somewhere in cluster to mount.

I also don't know how to do it. I was just asking about implementation and didn't want to force any implementation. The goal is to make it easy to support.